internal/v2/fakes2av2/fakes2av2_test.go

```diff
@@ -253,7 +253,7 @@ func TestSetUpSession(t *testing.T) {
 				ReqOneof: &s2av2pb.SessionReq_OffloadPrivateKeyOperationReq {
 					&s2av2pb.OffloadPrivateKeyOperationReq {
 						Operation: s2av2pb.OffloadPrivateKeyOperationReq_SIGN,
-						SignatureAlgorithm: s2av2pb.SignatureAlgorithm_S2A_SSL_SIGN_RSA_PSS_RSAE_SHA256,
+						SignatureAlgorithm: s2av2pb.SignatureAlgorithm_S2A_SSL_SIGN_RSA_PKCS1_SHA256,
```

Comment:
Should we also add a test case that uses the s2av2pb.SignatureAlgorithm_S2A_SSL_SIGN_RSA_PSS_RSAE_SHA256 signature algorithm, so that we cover both codeblocks?

Reply:
I have a test case for this code block in the remote_signer_test.go file. 

The reason I didn't include it here is because of the structure of the tests in this file, we compare the proto responses. Since a PSS signature is not deterministic(salt is determined in the [SignPSS](https://cs.opensource.google/go/go/+/refs/tags/go1.18.3:src/crypto/rsa/pss.go;l=264;bpv=0) function), there isn't a way to compare the signed bytes directly. The way that I verify the PSS signature in remote_signer_test.go is by calling [VerifyPSS](https://cs.opensource.google/go/go/+/refs/tags/go1.18.3:src/crypto/rsa/pss.go;l=290;bpv=0). 

I could do the same thing I did in remote_signer_test.go here, by creating a separate function to verify this branch, to get full code coverage

Reply:
SG as long as its tested in remote_signer_test.go, that's fine.